src/stories/components/Tabs.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import { Meta, Story, Props, ArgsTable, Canvas, Preview } from '@storybook/addon-docs/blocks';
+import Tabs from '../../components/Tabs.vue';
+
+<Meta
+	title="Componentes/Navegação/Tabs"
+	component={ Tabs }
+	argTypes={{
+		activeColor: {
+			control:{
+				type: 'select',
+				options: [
+					'turquoise',
+					'green',
+					'blue',
+					'indigo',
+					'violet',
+					'pink',
+					'red',
+					'orange',
+					'amber',
+					'gray',
+				],
+			}
+		},
+	}}
+/>
+
+export const Template = (args, { argTypes }) => ({
+	props: Object.keys(argTypes),
+	components: { Tabs },
+	template: /*html*/ `
+		<div style="background-color: #F8F9FA; padding: 16px; border-radius: 4px;">
+			<cds-tabs
+				v-bind="$props"
+				@right-click="handle"
+				@add-action="add"
+			>
+				<template v-slot:room-42>
+					Sala de Psicologia
+				</template>
+				<template v-slot:room-51>
+					Sala de Fisioterapia
+				</template>
+				<template v-slot:room-13>
+					Sala de Otorrinolaringologista
+				</template>
+			</cds-tabs>
+	</div>
+	`,
+	methods: {
+		handle(e) {
+			console.log('right-click', e);
+		},
+		add(e) {
+			console.log('add', e);
+		},
+	},
+});
+
+export const template = /*html*/ `
+	<cds-tabs 
+		:tabs="tabs"
+		:active-tab="activeTab"
+	>
+		<template v-slot:tab-1>
+			Conteúdo da tab-1
+		</template>
+		<template v-slot:tab-2>
+			Conteúdo da tab-2
+		</template>
+		<template v-slot:tab-3>
+			Conteúdo da tab-3
+		</template>
+	</cds-tabs>
+	`;
+
+export const docsDecorator = () => {
+	return {
+		data() {
+			return {
+				template,
+			};
+		},
+		template: /*html*/ `
+			<source-code-wrapper
+				:exampleSourceCode="template"
+			>
+				<story slot="component-preview"/>
+			</source-code-wrapper>`,
+	};
+};
+
+# Tabs
+
+### Tabs são componentes utilizados para criar soluções de navegação interna.
+---
+<br />
+
+## Recomendamos o uso de Tabs quando:
+- For necessário separar o conteúdo por abas e navergar por elas.
+- Não for o caso de utilizar soluções como navbars, sidebars, cards ou links como navegação.
+
+<br />
+
+## Não recomendamos o uso de Tabs quando:
+- Houver muitos itens a serem mostrados como abas.
+- For necessário utilizar roteamento ou links externos.
```

Comment:
Não entendi bem essa recomendação

Reply:
É por que se for trocar de página de alguma forma, pelo Router ou por links externos não deve ser usado ele e sim nav-bar ou similares

Reply:
@johnn1sbo3s sugestão de como pode melhorar essa recomendação?

Reply:
Algo do tipo "quando a ação de clique realizada pelo usuário o levar para outra página (outro contexto)".